doc/ranger.pod

```diff
@@ -1170,6 +1170,10 @@ Sets the state for the version control backend. The possible values are:
  local      display only local state.
  enabled    display both, local and remote state. May be slow for hg and bzr.
 
+=item vcs_msg_length [int]
+
+Truncate the long commit messages to this length when shown in the statusbar.
```

Comment:
Maybe mention that only the first line is shown?

Reads a bit meh to me. How about, "Length to truncate first line of commit messages to when shown in the statusbar. Defaults to 50."?

Reply:
Fair enough.